static/src/javascripts/projects/common/modules/commercial/adblock-messages.js

```diff
@@ -31,23 +31,19 @@ define([
         return config.switches.adblock;
     }
 
-    function noAdblockMsg() {
```

Comment:
I did a find-and-replace across the full codebase - only the test was using this function.